apps/docs/components/ui/sidebar/Sidebar.tsx

```diff
@@ -4,14 +4,71 @@ import Link from "next/link";
 import cx from "classnames";
 import { usePathname } from "next/navigation";
 import getPageLinks from "@/utils/getPageLinks";
+import Overlay from "@/components/ui/overlay/Overlay";
+import React, { useEffect, useRef } from "react";
```

Comment:
order of import https://gsoftdev.atlassian.net/wiki/spaces/Workleap/pages/3686760494/Coding+Standard#Import-order